test/basic.jl

```diff
@@ -364,6 +364,16 @@ end
     end
 end
 
+@testset "repeat" begin
+    for (size, counts) in Iterators.product(
```

Comment:
```suggestion
    @testset for (size, counts) in Iterators.product(
```

If it fails we know which config failed